docs/source/user_guide/options.rst

```diff
@@ -0,0 +1,216 @@
+====================
+Options and settings
+====================
+
+Overview
+--------
+Koalas has an options system that lets you customize some aspects of its behaviour,
+display-related options being those the user is most likely to adjust.
+
+Options have a full "dotted-style", case-insensitive name (e.g. ``display.max_rows``).
+
+The API is composed of 3 relevant functions, available directly from the ``koalas``
+namespace:
+
+* :func:`~koalas.get_option` / :func:`~koalas.set_option` - get/set the value of a single option.
+* :func:`~koalas.reset_option` - reset one or more options to their default value.
```

Comment:
These functions are not linked. Seems like we can link with something like the following:

```diff
diff --git a/docs/source/user_guide/options.rst b/docs/source/user_guide/options.rst
index 783ca31..503c89d 100644
--- a/docs/source/user_guide/options.rst
+++ b/docs/source/user_guide/options.rst
@@ -1,6 +1,7 @@
 ====================
 Options and settings
 ====================
+.. currentmodule:: databricks.koalas

 Overview
 --------
@@ -12,8 +13,8 @@ Options have a full "dotted-style", case-insensitive name (e.g. ``display.max_ro
 The API is composed of 3 relevant functions, available directly from the ``koalas``
 namespace:

-* :func:`~koalas.get_option` / :func:`~koalas.set_option` - get/set the value of a single option.
-* :func:`~koalas.reset_option` - reset one or more options to their default value.
+* :func:`get_option` / :func:`set_option` - get/set the value of a single option.
+* :func:`reset_option` - reset one or more options to their default value.
```

Reply:
nice!